articles/errors/deprecation-errors.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Deprecation Error Reference
+description: A listing of errors and descriptions relating to deprecations.
+toc: true
+---
+# Deprecation Error Reference
+
```

Comment:
Can you add a couple of sentences here describing what this doc is about and set expectations? Past lessons have taught us that the header alone is not sufficient.